src/global.css

```diff
@@ -6,23 +6,23 @@
   font-family: 'UncutSans_Regular';
```

Comment:
I would make this `UncutSans` for all of the `@font-face`s, that way you don't need the change the font-family all the time.

Reply:
Sorry, I'm not sure if I follow. If I make them all the same `font-face` the fonts don't render correctly 🤔 
<img width="1117" alt="image" src="https://user-images.githubusercontent.com/1179291/200908476-fbcd8374-1be6-4d50-b491-56ce317c9616.png">

Reply:
Hmm, that's odd, that should work. I'll give it a go in a few hours 👀

Reply:
I made some changes, seems to be ok now 👍

Reply:
Sorry for the formatting commits, forgot to disable that 😅 
Let me know if I need to squash those commits and if it works for you.

Reply:
Thanks! That's an interesting fix 🤔 shouldn't it just default to the last @font-face defined in the list if they all have the same name? https://github.com/webnative-examples/webnative-app-template/pull/87/files#diff-6a7ee04012624d3db7b781ffd397b00ae6dff2154c43fc5f20624ebc0fdb4b5aR22

Reply:
I've never used multiple font-faces that all have the same name, but point to different files. I'll try to find some documentation on this 👀